presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1190,7 +1190,7 @@ private PhysicalOperation visitScanFilterAndProject(
             // SYSTEM sampling is performed in the coordinator by dropping some random splits so the SamplingNode can be skipped here.
             else if (sourceNode instanceof SampleNode) {
                 SampleNode sampleNode = (SampleNode) sourceNode;
-                checkArgument(sampleNode.getSampleType() == SampleNode.Type.SYSTEM, format("%w sampling is not supported", sampleNode.getSampleType()));
+                checkArgument(sampleNode.getSampleType() == SampleNode.Type.SYSTEM, format("%s sampling is not supported", sampleNode.getSampleType()));
```

Comment:
`checkArgument` takes `(b, format, ...args)` so `format(` call is redundant here